app/src/main/java/com/concordium/wallet/ui/passphrase/setup/PassPhraseViewModel.kt

```diff
@@ -67,5 +70,14 @@ class PassPhraseViewModel(application: Application) : AndroidViewModel(applicati
             generatedPhrase(),
             password
         )
+        saveSeedPhrase()
```

Comment:
To avoid the risk of `saveSeedPhrase` storing something other than `tryToSetEncryptedSeedPhrase`, you should refactor it to something like:
```
val generatedPhrase = generatedPhrase();
_saveSeedLiveData.value = AuthPreferences(getApplication()).tryToSetEncryptedSeedPhrase(
            generatedPhrase,
            password
)
saveSeedPhrase(generatedPhrase)
```